payment/src/routes/__test__/create-payment.test.ts

```diff
@@ -125,8 +125,8 @@ it("returns a 201 with valid inputs", async () => {
   });
   await product.save();
 
-  const itemsPrice = parseFloat(product.price.toFixed(2));
-  const taxPrice = parseFloat((product.price * 0.07).toFixed(2));
+  const itemsPrice = Math.floor(parseFloat(product.price.toFixed(2)));
+  const taxPrice = Math.floor(parseFloat((product.price * 0.07).toFixed(2)));
```

Comment:
Sometimes create-payment test produces `400 Bad Request` because stripe API treated the float amount of money as an error.
It comes from `taxPrice` not rounded before addition with `itemsPrice` in `totalPrice`.

Reply:
`Math.floor()` method will fix this from happening.